### ISSUE TITLE
Tag colours not changed by theme in some databases

### ISSUE THREAD

**Issue by blorbb** [notion-enhancer/repo#35]
**What is happening?**

The list and timeline databases are showing default tag colours.

The gruvbox-dark theme is also showing no tag colours in any of the databases except for gray and light gray. Default colours still show on list and timeline database.

**What should be happening?**

Tags should be coloured according to the theme

**Screenshots (if applicable):**

Gruvbox dark: List (and timeline) shows default colours
![image](https://user-images.githubusercontent.com/88137137/147869703-2e2eb704-ed6e-4ba2-a9ff-0790567c27df.png)

Gruvbox dark: all other databases show no tag colours except for gray and light gray
![image](https://user-images.githubusercontent.com/88137137/147869718-5899c049-858d-4c3d-82b1-b4a01ce655d2.png)

Other themes: default colours in timeline and list databases
![image](https://user-images.githubusercontent.com/88137137/147869743-299c19a9-3683-49b2-8349-3c053513af5b.png)

The colours show properly on the table next to the timeline. (not in screenshot)

What should be happening: colours properly applied
![image](https://user-images.githubusercontent.com/88137137/147869814-2ab30978-d136-4b73-8b93-4b56e25bd88e.png)

**Operating system/browser name & version:**

Windows 10 21H1

**notion-enhancer and/or notion-repackaged version:**

Notion repackaged v0.11.0

**notion-enhancer configuration (enabled/disabled mods and their settings):**

- Pastel dark/Gruvbox dark
- Icon sets integration
- Tweaks: thicker bold text, wrap tables to page width, hide "Type '/' for commands"
- Integrated titlebar
- Tray
- Tabs
- Outliner
- Scroll to top
- Indentation lines
- Simpler databses
- Topbar icons: all on
- Truncated titles

**Comment by nicocossiom** [comment]
Same here, colors colored tags don't show properly when using Gruvbox theme, only displays the text. 
![With Gruvbox theme ](https://user-images.githubusercontent.com/72339792/148425194-6df83e36-5ac5-4c20-86eb-fae4897baf41.png)
![With different theme](https://user-images.githubusercontent.com/72339792/148425362-dcc77635-1985-4d35-bcf0-7060efce4fcf.png)

**Comment by Vermoot** [comment]
Adding my voice to this, I absolutely love the gruvbox dark theme but I'd like to get tag colours back!

**Comment by drneavin** [comment]
Just want to check whether this will fix the same issue with other dark themes? For example dracula theme is showing the default colors for tags and not the popped, lighter tag colors that it should show.

**Comment by dragonwocky** [comment]
@drneavin yes, all themes work on top of the same system, so when this is fixed it will be fixed for all themes
